homeassistant/components/foscam/config_flow.py

```diff
@@ -28,14 +32,15 @@
         vol.Required(CONF_USERNAME): str,
         vol.Required(CONF_PASSWORD): str,
         vol.Required(CONF_STREAM, default=STREAMS[0]): vol.In(STREAMS),
+        vol.Required(CONF_RTSP_PORT, default=DEFAULT_RTSP_PORT): int,
     }
 )
 
 
 class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a config flow for foscam."""
 
-    VERSION = 1
+    VERSION = 2
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
     async def _validate_and_create(self, data):
```

Comment:
Can we check if the user has entered the same host and port as an existing camera config entry, and show an error to the user in that case?

We can iterate all current config entries with `hass.config_entries.async_entries`.

Reply:
Oh, that would be nice indeed I tried to jeep the already configured functionallity but didnt managed to do it. I'll do that.

Reply:
Done!